apps/explorer/src/app/routes/network-parameters/network-parameters.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
follow up (non-blocking): we should be consistent in our naming of test files, some are .test.ts some are .spec.ts.